bundles/org.palladiosimulator.probeframework/src/org/palladiosimulator/probeframework/probes/ProbeConfigurations.java

```diff
@@ -0,0 +1,61 @@
+package org.palladiosimulator.probeframework.probes;
+
+import java.util.Optional;
+
+/**
+ * This class contains factory methods and identifier for common
+ * <code>ProbeConfiguration</code>s.
+ * 
+ * @author Sebastian Krach
+ *
+ */
+public final class ProbeConfigurations {
+    /** The sole probe contained in a configuration */
+    public static final String SINGULAR_PROBE = "org.palladiosimulator.probeframework.probes.singular";
```

Comment:
Might be better to use an enum. This set of constants is not extensible from outside anyways because the factory methods do not extensibility mechanisms.

Reply:
I added a comment to explain the extensibility concept. Additionally, I provided a factory method for custom sets.